src/pages/Main/Banners/Banners.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import BannerList from './BannerList';
+
+const Banners = props => {
+  const [currentSlide, setCurrentSlide] = useState(1);
+
+  const handleSlide = (e, direction) => {
```

Comment:
```suggestion
  const handleSlide = (direction) => {
```
생각해보니 e 사용하지 않는데 받을 필요 있나요? 이렇게 사용하거나 아니면 data-set 속성 사용하여 direction을 버리고 e만 사용해야 할거 같습니다

Reply:
말씀해주신대로 불필요한 e 삭제하고 direction 만 받도록 변경하였습니다!